### ISSUE TITLE
False Positive? `dialog[open]` triggers an error

### ISSUE THREAD

**Issue by thomas-price** [ryo-manba/stylelint-plugin-require-baseline#4]
Thank you for creating this package.

Currently, `dialog[open]` triggers an error: 
`Selector "open" is not a newly available baseline feature. (plugin/require-baseline)` 
for the CSS:
```
.my-dialog[open] {
	opacity: 1;

	@starting-style {
		opacity: 0;
	}
}
```
Do I correctly understand that [web-features](https://github.com/web-platform-dx/web-features) advises `dialog[open]` should not trigger an error hence, this is a false positive?

regards
